test/specs/accounts/create.account.spec.js

```diff
@@ -1,54 +1,40 @@
 'use strict';
-let config = require('../../../config.json');
-let loginPage = require('../../pages/login.page');
+let login = require('../../pages/login.page');
 let topSideBar = require('../../pages/topsidebar.page');
-let content = require('../../pages/accounts/acountContent.page');
-let form = require('../../pages/accounts/accountForm.page');
-let expect = require('chai').expect;
+let content = require('../../pages/content.page');
+let form = require(`../../pages/accounts/accountForm.${theme}.page`);
+
 describe('Acceptance Test to create new Account', function () {
-    this.retries(3);
 
     let accountFullData = {
         name: 'Account Sample',
         number: 100,
         site: 2,
-        parent: 'Edge Communications',
-        type: 'Other',
         industry: 'Agriculture',
         annualRevenue: 200,
         rating: 'Hot',
-        phone: 591591591,
-        fax: 591591222,
-        website: 'www.website.com',
-        tickerSymbol: 'ACC',
-        ownership: 'Private',
-        employees: 10,
-        sicCode: 200200200
+        phone: 591591591
+    };
+    let accountRequiredField = {
+        name: 'Account Required'
     };
 
     // Login application.
     beforeEach(function () {
-        loginPage.login(config.username, config.password);
-    });
-
-    // Delete an accounts created.
-    afterEach(function () {
```

Comment:
maybe you will need this afterEach to delete account hast been created before.

Reply:
I am working on this